src/WebUI/Components/Common.scss

```diff
@@ -26,3 +26,12 @@
 .secondary-text {
     color: $secondary-text;
 }
+
+.kube-status-container {
```

Comment:
unrelated comment: this is included in more than one scss file, that is this code is bloated, we shoudl see how this can be included only once in the end CSS

Reply:
are you talking about this class: '.kube-status-container' ?